packages/alert/src/Alert.tsx

```diff
@@ -0,0 +1,91 @@
+import CheckIcon from '@littlespoon/icons/lib/CheckIcon'
+import ExclamationIcon from '@littlespoon/icons/lib/ExclamationIcon'
+import colors from '@littlespoon/theme/lib/colors'
+import type React from 'react'
+
+import {
+  AlertActionLink,
+  AlertCloseIcon,
+  AlertDescription,
+  AlertMessages,
+  AlertTitle,
+  AlertWrapper,
+} from './AlertBase'
+
+export interface AlertProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The title of the component.
+   */
+  title?: string
+
+  /**
+   * The description of the component.
+   */
+  description?: string
+
+  /**
+   * The action link of the component.
+   */
+  actionLinkUrl?: string
+
+  /**
+   * The action link text of the component.
+   */
+  actionLinkText?: string
+
+  /**
+   * The action link text of the component.
+   */
+  dismissable?: boolean
```

Comment:
Can we remove this prop? Can we infer if the alert is dismissable if an `onClose` prop is passed?

Reply:
updated